assertj-tests/assertj-integration-tests/assertj-core-tests/src/test/java/module-info.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2025 the original author or authors.
+ */
+open module org.assertj.tests.core {
+  requires com.fasterxml.jackson.core;
+  requires com.fasterxml.jackson.databind;
+  requires com.google.common;
+  requires hamcrest.core;
+  requires java.sql;
+  requires javax.servlet.api;
+  requires junit;
+  requires net.bytebuddy;
+  requires nl.jqno.equalsverifier;
+  requires org.apache.commons.lang3;
+  requires org.assertj.core;
+  requires org.junit.jupiter.params;
+  requires org.junitpioneer;
+  requires org.mockito;
```

Comment:
Mockito 5.16.0 now ships module descriptors 🎉

Reply:
Thanks for the hint! I'll update and let you know 👍

Reply:
@TimvdLippe It seems there are some errors in our integration tests when upgrading Mockito from 5.15.2 to 5.16.0.

I'll dig a bit more and raise an issue with the details.